lib/Doctrine/DBAL/Driver/Statement.php

```diff
@@ -19,6 +19,8 @@
 
 namespace Doctrine\DBAL\Driver;
 
+use Doctrine\DBAL\DBALException;
```

Comment:
You're saying that even if some of the methods throw exceptions, the contract implies they shouldn't? I thought the opposite.